sorc/ncep_post.fd/WRFPOST.f

```diff
@@ -756,13 +756,13 @@ PROGRAM WRFPOST
             STOP 9999
 
           END IF 
-        ELSE IF(TRIM(IOFORM) == 'sigio')THEN 
-          IF(MODELNAME == 'GFS') THEN
-            CALL INITPOST_GFS_SIGIO(lusig,iunit,iostatusFlux,iostatusD3D,idrt,sighead)
-          ELSE
-            PRINT*,'POST does not have sigio option for this model, STOPPING'
-            STOP 99981		
-          END IF 	
+!       ELSE IF(TRIM(IOFORM) == 'sigio')THEN 
+!         IF(MODELNAME == 'GFS') THEN
+!           CALL INITPOST_GFS_SIGIO(lusig,iunit,iostatusFlux,iostatusD3D,idrt,sighead)
+!         ELSE
+!           PRINT*,'POST does not have sigio option for this model, STOPPING'
+!           STOP 99981		
+!         END IF 	
```

Comment:
@hertneky To remove sigio capability, do you think line 252. 152, 157 and 591 - 657 can be removed?

Reply:
@WenMeng-NOAA Yes these certainly can be removed. An oversight on my end since I was focused on a successful build without the sigio initpost routine. I will do this and then rerun tests on Hera, once it is back up.

Is it best in this case to remove rather than comment out?

Reply:
@hertneky Yes, deleting would be better if the regression tests. Also consider to remove sigio module under modulefiles directory (e.g. in wcoss_dell_p3).

Reply:
@WenMeng-NOAA Are the modulefiles under 'modulefiles/post' or 'modulefiles/upp' used? Should I remove sigio from those as well?

Reply:
@hertneky The changes are not needed for files under modulesfile/post and modulefile/upp which are legacy files for GFS V16. I will removed them in the future.